tests/python/frontend/tflite/test_forward.py

```diff
@@ -567,6 +567,24 @@ def test_forward_inception_v4_net():
     tvm.testing.assert_allclose(np.squeeze(tvm_output[0]), np.squeeze(tflite_output[0]),
                                 rtol=1e-5, atol=1e-5)
 
+#######################################################################
+# SSD Mobilenet
+# ---------
+
+def test_forward_ssd_mobilenet_v1():
+    """Test the SSD Mobilenet V1 TF Lite model."""
+    # SSD MobilenetV1
+    tflite_model_file = tf_testing.get_workload_official(
+        "https://neo-ai-dlr-test-artifacts.s3-us-west-2.amazonaws.com/tflite-models/ssd_mobilenet_v1_coco_2018_01_28_nopp.tgz",
```

Comment:
Move this to our dmlc / web-data : https://github.com/dmlc/web-data

Reply:
Or download the model directly from tf lite community if possible

Reply:
TF provides tflite files only for quantized models, but TVM does not support quantized models yet.
For non-quantized models they provide `tflite_graph.pb` file which is compatible with `tflite_convert` util and can be converted to tflite file.

For now I can put my tflite converted file to dmlc/web-data.

I can also investigate if Jenkins worker has `tflite_convert` util or python module `tensorflow.lite.TFLiteConverter` available. In that case I can download tgz model file from Tensorflow zoo, unpack it and convert `tflite_graph.pb` to tflite file.

Reply:
Actually, official non-quantized tar.gz ssd model files do not have `tflite_graph.pb`. They only have inference pb file and a checkpoint.
tflite_graph.pb can be created from checkpoint using `object_detection/export_tflite_ssd_graph.py` from tensorflow/models repo.
We do not have this repo and util on Jenkins worker.

I opened PR to add `ssd_mobilenet_v1_coco_2018_01_28_nopp.tgz` to `dmlc/web-data`
https://github.com/dmlc/web-data/pull/186